docs/references/protocol/transactions/types/ammclawback.md

```diff
@@ -0,0 +1,70 @@
+# AMMClawback
+
+<!--> Add link to github source after feature is merged into devnet. <-->
+[[Source]](https://github.com/XRPLF/rippled/blob/master/src/xrpld/app/tx/detail/AMMClawback.cpp "Source")
+
+_(Added by the [AMM amendment][])_
+
+Claw back tokens from a holder that has deposited your issued tokens into an AMM pool.
+
+Clawback is disabled by default. To use clawback, you must send an [AccountSet transaction](https://xrpl.org/docs/references/protocol/transactions/types/accountset) to enable the **Allow Trust Line Clawback** setting. An issuer with any existing tokens cannot enable clawback. You can only enable **Allow Trust Line Clawback** if you have a completely empty owner directory, meaning you must do so before you set up any trust lines, offers, escrows, payment channels, checks, or signer lists. After you enable clawback, it cannot reverted: the account permanently gains the ability to claw back issued assets on trust lines.
+
+
+## Example {% $frontmatter.seo.title %} JSON
+
+```json
+{
+  "TransactionType": "AMMClawback",
+  "Account": "rPdYxU9dNkbzC5Y2h4jLbVJ3rMRrk7WVRL",
+  "Holder": "rvYAfWj5gh67oV6fW32ZzP3Aw4Eubs59B",
+  "Asset": {
+      "currency" : "FOO",
+      "issuer" : "rPdYxU9dNkbzC5Y2h4jLbVJ3rMRrk7WVRL"
+  },
+  "Asset2" : {
+      "currency" : "BAR",
+      "issuer" : "rHtptZx1yHf6Yv43s1RWffM3XnEYv3XhRg"
+  },
+  "Amount": {
+      "currency" : "FOO",
+      "issuer" : "rPdYxU9dNkbzC5Y2h4jLbVJ3rMRrk7WVRL",
+      "value" : "1000"
+  }
+}
+```
+
+
+{% raw-partial file="/docs/_snippets/tx-fields-intro.md" /%}
+
+
+| Field              | JSON Type | [Internal Type][] | Required | Description |
+|:-------------------|:----------|:------------------|:---------|:------------------|
+| `Account` | String | AccountID | Yes | The issuer of the asset being clawed back. This field must match the account submitting the transaction. |
+| `Asset`   | Object | STIssue   | Yes | Specifies the asset that the issuer wants to claw back from the AMM pool. In JSON, this is an object with `currency` and `issuer` fields. The `issuer` field must match with `Account`. |
+| `Asset2`  | Object | STIssue   | Yes | Specifies the other asset in the AMM's pool. In JSON, this is an object with `currency` and `issuer` fields (omit `issuer` for XRP). |
+| `Amount`  | [Currency Amount](https://xrpl.org/docs/references/protocol/data-types/basic-data-types#specifying-currency-amounts) | Amount | No | The maximum amount to claw back from the AMM account. The `currency` and `issuer` subfields should match the `Asset` subfields. If this field isn't specified, or the `value` subfield exceeds the holder's available tokens in the AMM, all of the holder's tokens will be clawed back. |
```

Comment:
"...all of the holder's tokens are clawed back."

Also note that holder is lowercase here, but below we talk about the `Holder`. I think it should always be lowercase, unless we're referring to the `Holder` in the context of syntax. (for example, at the end of line 53 and on line 63, I'd use lowercase, while I agree that it is correctly formatted as syntax on line 67.